public/javascripts/data.js

```diff
@@ -262,7 +293,7 @@ var dataNS = odkmaker.namespace.load('odkmaker.data');
         {
 
             // The translation for the main control object obj
-            var schoolyard = [{
+            var tx = [{
```

Comment:
Rename variable because the pushing children across a schoolyard has no explanatory value.